tests/transports-integrations/tests/utils/config_loader.py

```
@@ -0,0 +1,309 @@
+                )
+
+        return True
+
+    def print_config_summary(self):
+        """Print a summary of the configuration"""
+        print("🔧 BIFROST INTEGRATION TEST CONFIGURATION")
+        print("=" * 80)
+
+        # Bifrost configuration
+        bifrost = self.get_bifrost_config()
+        print(f"\n🌉 BIFROST GATEWAY:")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)